docs/painless/painless-api-reference/painless-api-reference-string-sort/packages.asciidoc

```diff
@@ -0,0 +1,89 @@
+// This file is auto-generated. Do not edit.
+
+
+[role="exclude",id="painless-api-reference-string-sort-org-elasticsearch-xpack-eql-expression-function-scalar-whitelist"]
+=== String Sort API for package org.elasticsearch.xpack.eql.expression.function.scalar.whitelist
+See the <<painless-api-reference-string-sort, String Sort API>> for a high-level overview of all packages and classes.
+
+[[painless-api-reference-string-sort-InternalEqlScriptUtils]]
+==== InternalEqlScriptUtils
+* static String between(String, String, String, Boolean, Boolean)
+* static Boolean cidrMatch(String, List)
+* static String concat(List)
+* static Boolean endsWith(String, String, Boolean)
+* static Integer indexOf(String, String, Number, Boolean)
+* static Integer length(String)
+* static Number number(String, Number)
+* static String string(Object)
+* static Boolean stringContains(String, String, Boolean)
+* static String substring(String, Number, Number)
+* boolean {java11-javadoc}/java.base/java/lang/Object.html#equals(java.lang.Object)[equals](Object)
+* int {java11-javadoc}/java.base/java/lang/Object.html#hashCode()[hashCode]()
+* String {java11-javadoc}/java.base/java/lang/Object.html#toString()[toString]()
```

Comment:
Please exclude these as well. These really are only meant to be used internally.

Reply:
Excluded.